docs/src/test-projects-js.md

```diff
@@ -206,6 +205,61 @@ Running order:
 
 <img width="70%" style={{display: 'flex', margin: 'auto'}} alt="Browser login project is blue, database is red and e2e tests relies on both" loading="lazy" src="https://user-images.githubusercontent.com/13063165/225938262-33c1b78f-f092-4762-a478-7f8cbc1e3b21.jpg" />
 
+### Teardown
+
+You can also teardown your setup by adding a teardown property to your setup project. This will run after all dependencies have run.
```

Comment:
```suggestion
You can also teardown your setup by adding a teardown property to your setup project. This will run after all dependent projects have run.
```